src/main/java/com/blackducksoftware/integration/build/BuildDependency.java

```diff
@@ -18,90 +18,73 @@
  *******************************************************************************/
 package com.blackducksoftware.integration.build;
 
-import java.io.Serializable;
-import java.util.List;
-
-public class BuildDependency implements Serializable {
-	private static final long serialVersionUID = 7110967596618980400L;
+import java.util.HashSet;
+import java.util.Set;
 
+public class BuildDependency {
 	private String group;
 	private String artifact;
 	private String version;
+	private String id;
 	private String classifier;
-	private List<String> scope;
+	private Set<String> scopes = new HashSet<String>();;
 	private String extension;
+	private MatchType matchType = MatchType.UNKNOWNMATCH;
+	private String projectName;
+	private String versionName;
+	private String licenseName;
+	private VulnerabilityCounts vulnerabilityCounts = new VulnerabilityCounts();
 
-	public String getGroup() {
-		return group;
+	private void setId() {
+		if (null != group && null != artifact && null != version) {
```

Comment:
I assume then, that this is also an issue in the existing code for 2.0.2? The behavior of this code is at least attempting to be exactly what was done in 2.0.2.